src/pages/components/tile/usage.mdx

```diff
@@ -366,88 +401,95 @@ the chevron icon.
+#### Icon and call-to-action button
 
-![Example image of expandable tiles.](images/tile-usage-7.png)
+On clickable tiles, use the arrow icon with the `interactive` token in light
+themes and the `icon-primary` token for dark themes. Place the icon at the
+bottom right to indicate navigation. Avoid using interactive links or buttons
+that can be confusing as multiple click targets on clickable tiles.
 
-<Caption>Tiles get wider as the browser gets larger </Caption>
+<DoDontRow>
+  <DoDont type="do" caption="Do right align the icon when it is by itself">
 
-</Column>
-</Row>
+![Example of "do" on clickable tile](images/tile-usage-9.png)
 
-### Alignment
+  </DoDont>
+
+  <DoDont type="do" caption="Do right align the icon when there is an additional text">
 
-Always strive for left alignment whenver possible. Type in tiles should always
-be left aligned. Icons or icon buttons can be aligned to either left or right
-edge depending on the tiles’ complexity.
+![Example of "do" on clickable tile](images/tile-usage-10.png)
 
-For example: In situations where there are both an icon or pictogram with an
-icon button, place the icon or pictorgram in the lower left corner of the tile
-and move the icon button (action) to the right corner.
+  </DoDont>
+</DoDontRow>
 
 <DoDontRow>
-  <DoDont type="do" caption="Do left align icon, link, or text when it is by itself.">
 
-![Do left align icon, link, or text when it is by itself.](images/tile-usage-8.png)
+  <DoDont type="do" caption="Do right align the icon when there is an icon or pictogram">
+
+![Example of "do" on clickable tile](images/tile-usage-11.png)
 
   </DoDont>
 
-  <DoDont type="dont" caption="Do not right align an icon, link, or text when it is by itself">
+  <DoDont type="dont" caption="Do not add interactive links or components on clickable tiles">
 
-![Do not right align an icon, link, or text when it is by itself.](images/tile-usage-9.png)
+![Example of "do not" on clickable tile](images/tile-usage-12.png)
 
   </DoDont>
 </DoDontRow>
 
-<DoDontRow>
+On base tiles, place the interactive link at the bottom left or the button at
+the bottom to indicate call-to-action items.
 
-  <DoDont type="do" caption="Do move icon to the right when there is an icon or pictogram.">
+<DoDontRow>
+  <DoDont type="do" caption="Do left align the interactive link at the bottom on base tile">
 
-![Do move icon to the right when there is an icon or pictogram.](images/tile-usage-10.png)
+![Example of "do" on base tile](images/tile-usage-13.png)
 
   </DoDont>
 
-  <DoDont type="do" caption="Do move icon to the right when there is text or link.">
+  <DoDont type="do" caption="Do place the button at the bottom on base tile">
 
-![Do move icon to the right when there is text or link.](images/tile-usage-11.png)
+![Example of "do" on base tile](images/tile-usage-14.png)
 
   </DoDont>
 </DoDontRow>
 
-### Placement
-
 #### Tile groups
 
 Tile groups are a useful way of aligning tiles that have a strong relationship.
```

Comment:
Tile groups are useful for aligning tiles that have a strong relationship. Tile groups flow horizontally from left to right usually and have similar hierarchical importance as with navigation or catalog tiles.